widgets/mem_freebsd.lua

```diff
@@ -0,0 +1,67 @@
+----------------------------------------------
```

Comment:
These should be merged. I do not have a machine with swap available at the moment, so i need to  improvise a bit here. Anyone else is welcome, too. :)